routes.go

```diff
@@ -312,10 +322,11 @@ func fixV2Duration(getLogger ancla.GetLoggerFunc, config ancla.TTLVConfig, v2Han
 			// if this is v2, we need to unmarshal and check the duration.  If
 			// the duration is bad, change it to 5m and add a header. Then use
 			// the v2 handler.
-			logger := getLogger(r.Context())
+			logger := sallusthttp.Get(r)
 			if logger == nil {
-				logger = log.NewNopLogger()
+				panic("no logger supplied")
```

Comment:
you don't need to explicitly panic here.  sallust.Get is documented never to return nil.